packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -100,6 +106,69 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [onMouseUp, hideDropSelector],
   );
 
+  const handleKeyDown = React.useCallback(
+    event => {
+      switch (
+        event.keyCode // Begin switch 1
```

Comment:
```suggestion
        event.key // Begin switch 1
```

Reply:
Used event.code instead to be independent of different keyboard layouts.